pkg/apis/logging/v1/clusterlogging_types.go

```diff
@@ -192,6 +193,20 @@ type FluentdSpec struct {
 	Tolerations  []v1.Toleration   `json:"tolerations,omitempty"`
 }
 
+type LogFileMetricExporterSpec struct {
+        // The resource requirements for Fluentd
+        //
+        // +nullable
+        // +optional
+        Resources *v1.ResourceRequirements `json:"resources"`
+
+        // Define which Nodes the Pods are scheduled on.
+        //
+        // +nullable
+        NodeSelector map[string]string `json:"nodeSelector,omitempty"`
+        Tolerations  []v1.Toleration   `json:"tolerations,omitempty"`
```

Comment:
The `FluentdSpec` is already contains these fields, as these fields are required at pod level
So no need to duplicate them here, if `LogFileMetricExporterSpec` is only for container resources